sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3676,11 +3760,13 @@ pi_result piextQueueCreate(pi_context Context, pi_device Device,
     // Create as many command-lists as there are queues in the group.
     // With this the underlying round-robin logic would initialize all
     // native queues, and create command-lists and their fences.
-    PI_CALL(warmupQueueGroup(false, Q->ComputeQueueGroup.UpperIndex -
-                                        Q->ComputeQueueGroup.LowerIndex + 1));
+    auto InitialGroup = Q->ComputeQueueGroupsByTID.begin()->second;
```

Comment:
Please add a comment why you are only warming up first queue group here

Reply:
Done.